.github/README.md

```diff
@@ -0,0 +1,101 @@
+    <img src="https://dotnet.visualstudio.com/IronLanguages/_apis/build/status/ironpython3" alt="Azure CI Status" /></a>
+  <a style="text-decoration:none" href="https://gitter.im/IronLanguages/ironpython">
+    <img src="https://badges.gitter.im/IronLanguages/ironpython.svg" alt="Gitter" /></a>
+</p>
+
+**IronPython** is a popular, open-source implementation of Python 3.x for .NET that is built on top of its very own [Dynamic Language Runtime](https://github.com/IronLanguages/dlr).
+
+> [!NOTE]
+> There is still much that needs to be done to support Python 3. We are working on it, albeit slowly.
+> We welcome all those who would like to help! ❤️
+
+IronPython is an open-source implementation of the Python programming language that is tightly integrated with .NET. IronPython can use .NET and Python libraries, and other .NET languages can use Python code just as easily.
+
+IronPython 3 targets Python 3, including the re-organized standard library, Unicode strings, and all of the other new features.
+
+## 🎁 Installation
+
+Binaries of IronPython 3 can be downloaded from the [release page](https://github.com/IronLanguages/ironpython3/releases/latest), available in various formats: `.msi`, `.zip`, `.deb`, `.pkg`. The IronPython package is also available on [NuGet](https://www.nuget.org/packages/IronPython). See the [installation article](https://github.com/IronLanguages/ironpython3/wiki/Installing) for detailed instructions on how to install a standalone IronPython interpreter on various operating systems and .NET frameworks.
+
+## ✍️ Examples
+
+The following C# program:
+
+```cs
+using System.Windows.Forms;
+
+MessageBox.Show("Hello World!", "Greetings", MessageBoxButtons.OKCancel);
+```
+
+can be written in IronPython as follows:
+
+```py
+import clr
+clr.AddReference("System.Windows.Forms")
+from System.Windows.Forms import MessageBox, MessageBoxButtons
+
+MessageBox.Show("Hello World!", "Greetings", MessageBoxButtons.OKCancel)
+```
+
+Here is an example how to call Python code from a C# program.
+
+```cs
+var eng = IronPython.Hosting.Python.CreateEngine();
+var scope = eng.CreateScope();
+eng.Execute(@"
+def greetings(name):
+    return 'Hello ' + name.title() + '!'
+", scope);
+dynamic greetings = scope.GetVariable("greetings");
+System.Console.WriteLine(greetings("world"));
+```
+
+> This example assumes that `IronPython` has been added to the C# project as a NuGet package.
+
+More examples and documentation on how to use IronPython outside the interactive console can be found [here](https://github.com/IronLanguages/ironpython3/wiki/Using-IronPython).
+
+## 🐍 State of the Project
+
+The current target is Python 3.4, although features and behaviors from later versions may be included.
+
+See the following lists for features from each version of CPython that have been implemented:
+
+- [`Python3.0`](https://github.com/IronLanguages/ironpython3/wiki/WhatsNew%E2%80%90Python3.0)
```

Comment:
Bit weird that these are using the unicode hyphen instead of the good old ASCII dash. :smile:

Reply:
@slozier the ASCII dash symbol doesn't apply the bullet point in Markdown like the Unicode one does:

– ASCII dash
- Unicode dash

If you insist I'll add it though 🙂

Reply:
Sorry I wasn't clear. I meant in the filename `WhatsNew%E2%80%90Python3.0`.

Reply:
Oh! Right, I'll change that.